src/pretalx/submission/models/resource.py

```diff
@@ -32,6 +32,7 @@ class Resource(PretalxModel):
     description = models.CharField(
         null=True, blank=True, max_length=1000, verbose_name=_("Description")
     )
+    private = models.BooleanField(default=False, verbose_name=_("Non-public resource"))
```

Comment:
```suggestion
    is_private = models.BooleanField(default=False, verbose_name=_("Non-public resource"))
```

I’d prefer the `is_` naming convention for boolean fields (like with `Question.is_public` and `Event.is_public`). I know we don’t do this everywhere, but I feel like it’s less confusing during development.